lib/interviewer/selectors/interface.js

```diff
@@ -117,9 +118,15 @@ export const makeGetStageNodeCount = () => {
  * Return a filtered node list containing only nodes where node IDs contains the current promptId.
 */
 
+export const getPromptId = createSelector(
+  getPrompts,
+  getPromptIndex,
+  (prompts, promptIndex) => prompts[promptIndex].id,
+)
+
 export const getNetworkNodesForPrompt = createSelector(
   getNetworkNodesForType,
-  getPropPromptId,
+  getPromptId,
```

Comment:
Replace the old selector with the new one - this selector was previously also failing without us realizing.